enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -59,13 +59,19 @@ requests to off-cluster services, and those that use a
   + [Additional certificate authorities](https://docs.openshift.com/container-platform/4.12//rest_api/config_apis/proxy-config-openshift-io-v1.html#spec-trustedca) required to validate the proxy's certificate
 * Configure the proxy settings in WMCO-managed components on Windows nodes (kubelet, containerd runtime)
 * React to changes to the cluster-wide proxy settings during WMCO runtime
+* Synchronize environment variables `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` on Windows nodes with cluster-wide 
+  proxy in both shells _CMD_ and _PowerShell_
 * Maintain normal functionality in non-proxied clusters
 
 ### Non-Goals
 
 * First-class support/enablement of proxy utilization for user-provided applications
 * *ingress* and reverse proxy settings are out of scope
 * Monitor cert expiration dates or automatically replace expired CAs in the cluster's trust bundle
+* Windows workloads created in Windows nodes with cluster-wide proxy enabled do not inherit proxy settings from the 
+  node. This is the default behavior on Linux Nodes side as well.
+* PowerShell's shell sessions do not inherit proxy settings by default on Windows nodes with cluster-wide proxy enabled 
+  (See [Risks and Mitigations](#risks-and-mitigations) section for recommendations) 
```

Comment:
This link is broken for me, can you double-check?

Reply:
@mansikulkarni96  this works fine locally and I also tested it in https://github.com/jrvaldes/openshift-enhancements/blob/wmco-proxy-update/enhancements/windows-containers/windows-node-egress-proxy.md